projects/kit/components/input-slider/input-slider.template.html

```diff
@@ -45,7 +45,7 @@
         [keySteps]="computedKeySteps"
         [attr.disabled]="readOnly || disabled || null"
         [ngModel]="value"
-        (keyStepsInput)="safelyUpdateValue($event)"
+        (keyStepsInput)="onSliderChange($event)"
```

Comment:
Do you need to write any e2e or unit tests for cover this case?

Reply:
unfortunately, it is rather difficult because of 2 reasons:

- Cypress can emulate touch (but i require it to reproduce bug)
- Unit tests cannot trigger changing of form control via
```ts
function getSliderElement(): HTMLInputElement {
    return fixture.debugElement.query(By.css('input[type=range]')).nativeElement;
}

function changeSliderValue(newValue: string): void {
    const slider = getSliderElement();

    slider.value = newValue;
    slider.dispatchEvent(new Event('input'));
}
```